parsers/ada.c

```diff
@@ -624,17 +611,8 @@ static void readNewLine (void)
 		if (line == NULL)
 		{
 			lineLen = 0;
-			exception = EXCEPTION_EOF;
```

Comment:
Removing this mysterious loop is one of mankind's long-cherished wish like a world without war.
Thank you very much fo taking your time.

Reply:
Yeah, this was really mysterious when looking at it for the first time and I assumed the longjmp() was executed during normal parsing. But it turned out to be just a fail-safe mechanism if the parser started cycling somewhere and since none of the other parsers uses anything like that, better to avoid it.